object.c

```diff
@@ -4565,6 +4565,143 @@ InitVM_Object(void)
+     * - #sub:: Performs single substitution using the given pattern and either
+     *          a replacement string or a block.
+     *
+     * === \IO
+     *
+     * - #chomp:: Removes the trailing record separator, if any, from <tt>$_</tt>.
+     * - #chop:: Removes the last character, if any, from <tt>$_</tt>.
+     * - #gets:: Returns and assigns to <tt>$_</tt> the next line from the current input.
+     * - #open:: Creates an IO object connected to the given stream, file, or subprocess.
+     * - #p::  Writes the given objects to standard output.
+     * - #pp:: Prints the given objects in pretty form.
+     * - #print:: Prints the given objects to standard output.
+     * - #printf:: Prints the string resulting from applying the given format string
+     *             to any additional arguments.
+     * - #putc:: Equivalent to <tt.$stdout.putc(object)</tt> for the given object.
+     * - #puts:: Equivalent to <tt>$stdout.puts(*objects)</tt> for the given objects.
+     * - #readline:: Equivalent to Kernel.gets, except that +readline+
+     *               raises an exception at end of file.
+     * - #readlines:: Returns an array of the remaining lines from the current input.
+     * - #select:: From arrays of IO streams, returns the first one to become ready.
+     *
+     * === Procs
+     *
+     * - #lambda:: Returns a new Proc formed from the given block.
+     * - #proc:: Returns a new Proc; equivalent to Proc.new.
+     *
+     * === Tracing
+     *
+     * - #set_trace_func:: Sets the given proc as the handler for tracing,
+     *                     or disables tracing if given +nil+.
+     * - #trace_var:: Specifies the tracing of assignments to the given global variable.
+     * - #untrace_var:: Disables tracing of assignments to the given global variable.
+     *
+     * === Subprocesses
+     *
+     * - #fork:: Creates a subprocess that executes the given block, if any.
+     * - #spawn:: Executes the given command and returns its pid without waiting.
+     *
+     * === Loading
+     *
+     * - #autoload:: Registers the given file to be loaded when the given module
+     *               is first referenced.
+     * - #load:: Loads and executes the Ruby program found in the given file.
+     * - #require:: Loads the given file.
+     * - #require_relative:: Loads the given relative file.
+     *
+     * === Yielding
+     *
+     * - #tap:: Yields +self+ to the given block; returns +self+.
+     * - #then:: Yields +self+ to the block and returns the result of the block.
+     * - #yield_self:: Yields +self+ to the block and returns the result of the block.
+     *
+     * === \Random Values
+     *
+     * - #rand:: Returns a pseudo-random floating point number
+     *           strictly between 0.0 and 1.0.
+     * - #srand:: Seeds the system pseudo-random number generator with the given number.
+     *
+     * === Other
+     *
+     * - #callcc:: Generates a Continuation object, which it passes
```

Comment:
This isn't defined by default (you need to require `continuation`), so it shouldn't be documented.